Drop-In/src/main/java/com/braintreepayments/api/DropInActivityContract.java

```diff
@@ -0,0 +1,45 @@
+package com.braintreepayments.api;
+
+import static android.app.Activity.RESULT_FIRST_USER;
+import static android.app.Activity.RESULT_OK;
+
+import static com.braintreepayments.api.DropInClient.EXTRA_AUTHORIZATION;
+import static com.braintreepayments.api.DropInClient.EXTRA_CHECKOUT_REQUEST;
+import static com.braintreepayments.api.DropInClient.EXTRA_CHECKOUT_REQUEST_BUNDLE;
+import static com.braintreepayments.api.DropInClient.EXTRA_SESSION_ID;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.activity.result.contract.ActivityResultContract;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+
+/**
+ * A contract used to launch a {@link DropInActivity} with a {@link DropInClient} input and {@link DropInResult2} output.
```

Comment:
Is `contract` common terminology for this sort of flow on Android?

Reply:
I haven't seen it before, but I based this doc string on the class it extends (`ActivityResultContract`), which uses the `contract` terminology.